enterprise/admin/views.py

```diff
@@ -174,7 +176,8 @@ def get_form_view(self, request, customer_uuid, additional_context=None):
         render the form with appropriate context.
         """
         context = self._build_context(request, customer_uuid)
-        context.update(additional_context)
+        if additional_context:
+            context.update(additional_context)
```

Comment:
This fixes a bug when the caller of `get_form_view()` doesn't pass `additional_context`.